File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/controller/ActuatorController.java
package org.javamaster.invocationlab.admin.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author yudong
 */
@RestController
@RequestMapping("/actuator")
public class ActuatorController {

    @ResponseBody
    @GetMapping("/health")
    public String health() {
        return "{\"status\":\"UP\"}";
    }

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/controller/ActuatorController.kt
package org.javamaster.invocationlab.admin.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.ResponseBody
import org.springframework.web.bind.annotation.RestController

/**
 * @author yudong
 */
@RestController
@RequestMapping("/actuator")
class ActuatorController {

    @ResponseBody
    @GetMapping("/health")
    fun health(): String {
        return "{\"status\":\"UP\"}"
    }

}
